### DISCUSSION TITLE
How to specify tags when doing `klog start`?

### DISCUSSION THREAD

**Discussion by ITiot23** [jotaen/klog#292] [Help / Question] [answered]
A simple question, but i don't make it work. Wenn I create a new entry wirth `klog start #Tag` the time start but the Tag will not be transfered into the File. Enviroment is Windows Powershell. I'm doing something wrong?

**Comment by jotaen** [comment, marked as answer]
Tags are part of the summary, so you’d have to specify the `--summary` flag:

```
klog start --summary '#tag'
```

Or, for short:

```
klog start -s '#tag'
```

Note that you have to wrap the entire summary text into single quotes, otherwise your shell might treat the tag as shell comment and ignore it (due to the `#` character, which denotes a comment on shell).

**Reply by ITiot23** [reply]
Thank your for the quick and helpful answer. Just started to use klog, do you have a list with the parameters and its options? It's a very helpful tool and I recommend ist to my kolleges. Thank you for sharing it

**Reply by jotaen** [reply]
You can run the `--help` flag to see all available flags. The help output is available on the `klog` base command, as well as on all sub commands.

```
klog --help
klog start --help
```

**Reply by ITiot23** [reply]
Thanks for this information. I knew about the help function in the command but not that it’s available in the sub commands. This works for me
